src/WebJobs.Script.WebHost/Controllers/FunctionsController.cs

```diff
@@ -84,6 +88,44 @@ public async Task<IActionResult> CreateOrUpdate(string name, [FromBody] Function
             }
         }
 
+        [HttpPost]
+        [Route("admin/warmup")]
+        [Authorize(Policy = PolicyNames.AdminAuthLevel)]
```

Comment:
Might consider allowing AdminAuthLevelOrInternal, unless we know that the caller will be sending a site token.